src/digital.h

```diff
@@ -29,6 +30,39 @@
 
 #define LED 13
 
+#define IS_IN_PORTD_RANGE(pin) pin <= 7
```

Comment:
you may want to write it like that:
IS_IN_PORTD_RANGE(pin)    (pin <= 7)

same for the ones under

Reply:
Acutally, you want to write it like this:
```c
IS_IN_PORTX_RANGE(pin) ((pin) <= 7)
```
But you have a point here.